lib/index.js

```diff
@@ -4,6 +4,16 @@ var util = require('util');
 var EventEmitter = require('events').EventEmitter;
 
 /**
+ * routine errors
+ */
+var er = {
+  op              : 'nanomsg operation failed: ',
+  param           : 'parameter type err: ',
+  chan            : 'setting channel subscription string: ',
+  rmchan          : 'removing channel subscription string: ',
+}
```

Comment:
ya the need is all these additional calls to `new Error('...` are longer than 81 chars per line.

I'd rather one-line new Errors, but if you insist.

well, hey I had some other repeated errs in mind too, it's not just those.

eh, I can take it out I guess, but you'd really rather see calls to `new Error` broken into multiple lines, or worse: trailing off beyond 80 chars to the right of the screen?